airflow-core/src/airflow/ui/src/pages/Dashboard/Stats/DAGImportErrorsModal.tsx

```diff
@@ -84,6 +84,10 @@ export const DAGImportErrorsModal: React.FC<ImportDAGErrorModalProps> = ({ impor
             {visibleItems.map((importError) => (
               <Accordion.Item key={importError.import_error_id} value={importError.filename}>
                 <Accordion.ItemTrigger cursor="pointer">
+                  <Text display="flex" fontWeight="bold">
+                    {translate("Bundle name:")}
```

Comment:
Search for bundleName in the json files to fine the correct key to use here.

Is this only for 3.1? If not, we'll need to manually back port

Reply:
Done. It's only for 3.1

Reply:
addressed.